src/pybind/mgr/dashboard/frontend/src/app/core/navigation/api-docs/api-docs.component.ts

```diff
@@ -0,0 +1,26 @@
+import { HttpClient } from '@angular/common/http';
+import { Component, OnInit } from '@angular/core';
+
+import SwaggerUI from 'swagger-ui';
+
+@Component({
+  selector: 'cd-api-docs',
+  templateUrl: './api-docs.component.html',
+  styleUrls: ['./api-docs.component.scss']
+})
+export class ApiDocsComponent implements OnInit {
+  specUrl: any;
+
+  constructor(private http: HttpClient) {}
+
+  ngOnInit(): void {
+    this.http.get('/docs').subscribe((data: any) => {
+      this.specUrl = data;
```

Comment:
Is not simpler just to fetch the OpenAPI spec from its URL (`/docs/api-all.json`)?

In fact, I'd remove the `/docs` one and remove the api-all.json to something like `openapi.json`.

Reply:
Done. Thanks